src/pdf-image-source.cc

```diff
@@ -53,14 +53,23 @@ bool PDFImageSource::LoadAndScale(const DisplayOptions &opts, int frame_offset,
     const int max_display_page =
         (frame_count < 0) ? page_count
                           : std::min(page_count, start_page + frame_count);
+    PopplerRectangle selection;
```

Comment:
I'd probably call this variable `bounding_box` as `selection` typically means something else.

Reply:
(solved)